participant/tests/test_querysets.py

```diff
@@ -45,3 +45,60 @@ def test_when_user_is_not_user_instance(self):
             user="a string or not a user instance"
         )
         self.assertFalse(organization.exists())
+
+
+class ParticipantQuerysetTests(APITestCase):
+    def setUp(self):
+        organization = OrganizationFactory(name="this_test_organization")
+        self.participant1 = ParticipantFactory(organization=organization)
+        self.participant2 = ParticipantFactory(organization=organization)
+        self.participant3 = ParticipantFactory(organization=organization)
+
+        other_organization = OrganizationFactory(name="other_organization")
+        self.participant4 = ParticipantFactory(organization=other_organization)
+
+        self.user = UserFactory(
+            username="user_with_organization", organization=organization
+        )
+        self.superuser = UserFactory(
+            organization=organization, is_superuser=True, username="test_superuser"
+        )
+        self.user_without_participant = UserFactory(
+            username="user_without_organization"
+        )
+
+    def test_org_filter_user_with_participant(self):
+        participants = Participant.objects.filter_by_user(user=self.user)
+        self.assertEqual(3, participants.count())
+
+        organization_participants = [
+            self.participant1,
+            self.participant2,
+            self.participant3,
+        ]
+
+        for participant in organization_participants:
+            with self.subTest(participant_organization=participant.organization.name):
+                self.assertTrue(participant in participants)
```

Comment:
You'd want to check that using `filter_by_user()` actually returns what you'd expect.
⚠️ Please also note the function name change (I feel that would be more descriptive).

```python
def test_participant_filter_by_user_with_organisation(self):
    # Participant 1, 2 and 3 took the same test that belong to the Organisation of self.user
    expected_participants = [self.participant1, self.participant2, self.participant3]
    filtered_participants = Participants.objects.filter_by_user(user=self.user)

    self.assertEqual(expected_participants, list(filtered_participants))
```

Reply:
I already tried this:
def test_participant_filter_by_user_with_organisation(self):
    participants = Participant.objects.filter_by_user(user=self.user)

        organization_participants = [
            self.participant1,
            self.participant2,
            self.participant3,
        ]
        self.assertEqual(organization_participants, list(participants))

BUT IT SURPRISED ME WITH THE FOLLOWING:
AssertionError: Lists differ: [<Par[18 chars]ant: 6da72b3b-dff5-409e-9a25-5f83583af845 For:[175 chars] 39>] != [<Par[18 chars]ant: 34f66544-173c-48da-a990-a6f7fd87d973 For:[175 chars] 39>]

I actually copied it, to know WHY?

Reply:
I know this, but the question is why other code is working without sort.

Reply:
Oh no. That one is also not working now. Unlucky me. It worked only once and I put you the question. Random can be that bad I never know.

Reply:
sorted instances were not supported, so subtest was the most efficient. Or a count. Others are expensive.

Reply:
oh yeah, try this instead:
```python
def test_org_filter_user_with_participant(self):
    # Participant 1, 2 and 3 took the same test that belong to the Organisation of self.user
    expected_ids = [self.participant1.id, self.participant2.id, self.participant3.id]
    filtered_ids = Participant.objects.filter_by_user(user=self.user).values_list('id', flat=True)

    self.assertEqual(set(expected_ids), set(filtered_ids))
```

Reply:
good idea. if subtest be avoided